packages/core/src/combinator/merge.js

```diff
@@ -6,15 +6,14 @@ import Pipe from '../sink/Pipe'
 import IndexSink from '../sink/IndexSink'
 import { empty } from '../source/core'
 import { disposeAll, tryDispose } from '@most/disposable'
-import { copy, reduce } from '@most/prelude'
+import { reduce } from '@most/prelude'
 
 /**
- * @returns {Stream} stream containing events from all streams in the argument
- * list in time order.  If two events are simultaneous they will be merged in
- * arbitrary order.
+ * @returns {Stream} stream containing events from two streams in time order.
+ * If two events are simultaneous they will be merged in arbitrary order.
  */
-export function merge (/* ...streams*/) {
-  return mergeArray(copy(arguments))
+export function merge (stream1, stream2) {
+  return mergeArray([stream1, stream2])
```

Comment:
With these "And here 👍 " comments, were you saying they looked good or did you want me to rename stream1 & stream2? Sorry, just trying to follow along before I make the changes.